src/main/java/net/kodehawa/mantarobot/commands/CurrencyActionCmds.java

```diff
@@ -321,7 +321,7 @@ private static void mine(IContext ctx, Player player, MongoUser dbUser, Marriage
             var amount = 1 + random.nextInt(item.getDiamondIncrease());
             if (!player.fitsItemAmount(ItemReference.DIAMOND, amount)) {
                 message += "\n" + languageContext.get("commands.mine.diamond.overflow").formatted(amount);
-                overflowMoney += Math.round((ItemReference.DIAMOND.getValue() * 0.9) * amount);
+                overflowMoney += (int) Math.round((ItemReference.DIAMOND.getValue() * 0.9) * amount);
```

Comment:
To be fair, the explicit casting here is not needed. The result is the same. I imagine this is due to the linter complaining?

Reply:
yeah its just so it doesn't complain